src/vue/text-editor/text-editor.vue

```diff
@@ -57,9 +59,12 @@ export default {
       this.defineTextsShow = false
     }
   },
+  created () {
+    this.$textC.createAlignment()

```

Comment:
I decided to change previous approach and start from empty alignment.

Because creation alignment could start from various:
* updating origin/target on the screen, even changing direction or language
* from upload source texts (or created previously alignments)

This way to make business logic lighter here - we always start from an empty alignment, that could be filled, rewrite, saved and so on.

@balmas , do you have any thoughts against?

Reply:
I can't think of a strong argument against :-)